starter-code/routes/drones.js

```diff
@@ -1,21 +1,49 @@
 const express = require('express');
 
 // require the Drone model here
+const Drone = require('../models/drone');
 
 const router = express.Router();
 
 
-router.get('/drones', (req, res, next) => {
+router.get('/', (req, res, next) => {
   // Iteration #2
+  Drone.find({}, (err, data) => {
+    if (err) {
+      next();
```

Comment:
Use next for passing the error to the error handler:
```
next(err);
```
Same applies for the rest of next().